src/api/currency/content-types/currency/schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "kind": "collectionType",
+  "collectionName": "currencies",
+  "info": {
+    "singularName": "currency",
+    "pluralName": "currencies",
+    "displayName": "Financial.CurrencyConversion"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "year": {
+      "type": "string",
```

Comment:
**suggestion:** I think this should this be of type integer. Or is there a case were we would want something other than a number in this field?

Reply:
I see, that could be a problem.

Depending on how @jtfairbank feels about it it might make more sense to just use a date field instead on numbers for month and year?

Reply:
Created a 'big integer' for this field to resolve the issue of the comma in the number, and included max and min values for the year selection.